src/pages/Filing/PrivacyNotice.tsx

```diff
@@ -15,7 +15,7 @@ function PrivacyNotice(): ReactElement {
           <TextIntroduction
             heading='Privacy Notice'
             subheading='The Consumer Financial Protection Bureau (CFPB) is collecting data to test the functionality of the Small Business Lending Data Filing Platform.'
-            description='The testing will help ensure that the system can support the filing of data needed to enforce fair lending laws pursuant to Public Law No. 111-203, Title X, Section 1071, codified at 12 C.F.R. 1002. Participation is voluntary. If you choose to participate, the CFPB will collect your first and last name to test the "Complete your user profile" page. The CFPB will not use the data for any other purpose besides testing. Please do not submit any live data in the system.'
+            description='The testing will help ensure that the system can support the submission of data needed to enforce fair lending laws pursuant to Public Law No. 111-203, Title X, Section 1071, codified at 12 C.F.R. 1002. Participation is voluntary. If you choose to participate, the CFPB will collect your first and last name to test the "Complete your user profile" page. The CFPB will not use the data for any other purpose besides testing. Please do not submit any live data in the system.'
```

Comment:
Natalia and I chatted about leaving this "submission" since it was given to us for the privacy notice.